querygen/querygen.go

```diff
@@ -180,3 +180,26 @@ func GenReleaseCompletedQuery(index string) string {
 		String()
 	return q
 }
+
+// GenPullRequestCreatedQuery returns a Splunk query for generating Segment events
+// whenever a Pull request is created in the users GitHub repository.
+func GenPullRequestCreatedQuery(index string) string {
+	q, _ := NewUserJourneyQuery(index).
+		WithPredicate(
+			`verb=update `+
+				`"responseStatus.code"=200 `+
+				`"objectRef.apiGroup"="appstudio.redhat.com" `+
+				`"objectRef.resource"="components" `+
+				`"user.username"="system:serviceaccount:build-service:build-service-controller-manager" `+
+				`"responseObject.metadata.annotations.build.appstudio.openshift.io/status"="*pac*"
+					(NOT in "responseObject.metadata.annotations.build.appstudio.openshift.io/request"="*") `,
+		).
+		WithEventExpr(`"Pull request created"`).
+		WithCommands(
+			`spath input="responseObject.metadata.annotations.build.appstudio.openshift.io/status", path=pac.state output=build_status.pac.state`,
```

Comment:
What happened to the command that filters out events which have a non-enabled pac state?